Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM docker.io/library/golang:1.13 as builder
+FROM mirror.gcr.io/library/golang:1.15 as builder
```

Comment:
This is OK if we use the latest version, but we noticed that it may be withdrawn if a new version is created. It may be just an issue if a new version is released and anyone wants to build the operator with the previous version.
We can leave gcr mirror if we think that we should always use the latest version.

Reply:
Changed to docker.io/library/golang:1.15